torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -1001,11 +1000,12 @@ void StaticRuntime::display_nodes(
     // previous `run()` for memory planning of subsequent runs
     if (!planner_) {
       planner_ = std::make_unique<MemoryPlanner>(
-          this,
+          this->nodes(),
+          this->graph(),
+          this->outputs(),
           static_module_.values_share_same_storage(),
           static_module_.external_values(),
-          static_module_.opts().enable_out_variant,
-          static_module_.opts().optimize_graph_output_memory);
```

Comment:
unused in original (ctrl+f `manage_graph_output_memory` [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/static/impl.cpp#L1219).